Framework/Algorithms/src/CreateSampleWorkspace.cpp

```diff
@@ -159,6 +160,11 @@ void CreateSampleWorkspace::init() {
                   boost::make_shared<BoundedValidator<double>>(0, 1000, true),
                   "The distance along the beam direction from the source to "
                   "the sample in M (default:10.0)");
+  declareProperty("NumScanPoints", 1,
+                  boost::make_shared<BoundedValidator<int>>(0, 360, true),
```

Comment:
The detectors are rotated around the sample in 1 degree increments, hence the limit. Would ti be helpful to clarify this? It is only mentioned explicitly in the documentation.